src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-set">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="datebith"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-set">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          minlength="5"
+          maxlength="20"
+          required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+          type="password"
+          name="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+            type="radio"
+            name="catlike"
+            value="yes"
+            >
+            Yes
```

Comment:
You should only activate the radio buttons by clicking 'Yes' or 'No.' There is no need to activate them by clicking the text 'Do you love cats?'